contents/product-engineers/what-is-user-stickiness.md

```diff
@@ -0,0 +1,120 @@
+date: 2023-05-09
+title: 'What is user stickiness and how to measure it?'
+author:
+  - james-temperton
+featuredImage: >-
+  https://res.cloudinary.com/dmukukwp6/image/upload/posthog.com/contents/images/blog/posthog-marketing/marketing-hog.jpg
+featuredImageType: full
+tags:
+  - Product analytics
+  - Product metrics
+  - Product engineers
+  - Growth
+crosspost:
+  - Founders
+  - Blog
+---
+Can’t stop scrolling on TikTok? Endlessly gawping at Instagram? That’s stickiness. Popularized by Meta when it was still known as Facebook, the notion of stickiness remains a crucial measure of how engaging your product or service is regardless of whether you’re a planet-conquering social media app or a B2B SaaS company.
+
+In this guide we’ll explain what user stickiness is, why it’s so important, how to measure it, and give you some tips on how to increase the stickiness of your product.
+
+### What is user stickiness?
+
+Stickiness is a type of insight that lets you see how many people use your product on a regular basis. There are a few different ways to calculate it. DAU/MAU ratio is the most popular, but session frequency, session duration, and individual feature use can also help you understand how sticky your product is.
```

Comment:
I think key to this entire article is distinguishing stickiness, retention, and engagement. When should you use one vs the others?

This is an interesting take on stickiness focusing on DAU/MAU and how it can be deceiving: https://mixpanel.com/blog/what-is-a-good-stickiness-metric/

PostHog's stickiness insight looks different than DAU/MAU ratio though, why?

A potentially good definition that encompasses both:
"What percentage or amount of users engage with your app, website, or feature *frequently* (daily, weekly).